Source/engine/assets.hpp

```diff
@@ -238,6 +239,10 @@ inline bool ValidateHandle(const char *path, const AssetHandle &handle)
 }
 
 AssetRef FindAsset(const char *filename);
+inline AssetRef FindAsset(std::string_view filename)
+{
+	return FindAsset(filename.data());

```

Comment:
This is not technically correct because `filename.data()` is not guaranteed to be null-terminated.
I'll send a PR changing the argument to `std::string_view`.

Reply:
Ah, that's not possible because the underlying libmpq functions only support null-terminated C strings. We'd have to make lots of changes to libmpq first.

Reply:
I'll add the missing functions to libmpq

Reply:
Must've been thinking of `std::string::data()` when I wrote this.

Reply:
Done in https://github.com/diasurgical/devilutionX/pull/6489